src/components/ProgramLoader/Links.tsx

```
@@ -65,6 +65,27 @@ export const Links = () => {
           </div>
         </div>
       </li>
+      <li>
+        <div className="flex gap-2 text-[11px] mt-3">
+          <a href="https://github.com/polkadot-fellows/JIPs/blob/main/JIP-6.md" target="_blank">
```

Biome (2.3.13)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)